detox/scripts/build_universal_framework.sh

```diff
@@ -14,7 +14,7 @@ function remove_arch() {
 mkdir -p "${OUTPUT_DIR}"
 rm -fr "${OUTPUT_DIR}/${PROJECT_NAME}.framework"
 
-TEMP_DIR=$(mktemp -d -t DetoxBuild)
+TEMP_DIR=$(mktemp -d DetoxBuild.XXXX)
```

Comment:
But why can't we use `-t` flag? Is it missing in some version or what? Sorry, just fail to understand.

Reply:
@p3drosola, please respond.

Reply:
`-t` is completely different between the implementations. In GNU, it doesn't take a param and signals that the created temp file/dir should be under `$TMPDIR`, while on BSD it takes a param and is used for specifying a template name.

GNU:
> ‘-t’
> Treat template as a single file relative to the value of TMPDIR if available, or to the directory specified by -p, otherwise to ‘/tmp’. template must not contain slashes. This option is deprecated; the use of -p without -t offers better defaults (by favoring the command line over TMPDIR) and more flexibility (by allowing intermediate directories).

BSD:
> -t prefix
> Generate a template (using the supplied prefix and TMPDIR if set) to create a filename template.